README.md

```diff
@@ -1 +1,39 @@
 # Intention-Timer
+
+### Contributors
+
+Jani Palominu [janitastic](https://github.com/janitastic)
+
+Lauren Kessell [LKessell](https://github.com/LKessell)
+
+Andrey Bermudez [Andrey-1992](https://github.com/Andrey-1992)
+
+
+# About
+The Intention Timer app seeks to provide a chronometer to help the user to organize, achieve and keep tracking of their goals. With this app, each user will have the ability to set customize time frames and add specifications to work on, in three different categories:
+ **Study**
+ **Meditation**
+ **Exercises**
+
+
+ ## Technologies Used
+
+   Javascript
+
+   HTML
+
+   CSS
+
+# Instructions
+1. When the user first loads the app, they have to select one of the three categories to work on it.
+2. The user have to write a description about their activity during the timer.
+3. The user must specify in the section of minutes and second the time that he would like to work, after that he is ready so start the activity.
+4. In the next window he can verify is the information is correct, and get ready to set the timer on.
+5. When the users complete the activity, they would have the possibility to log it and keep track of their process.
+6. Finally they can start a new activity clicking the **CREATE A NEW ACTIVITY** button.
+
+   ### Deployed page:
+     [Intention Timer](https://andrey-1992.github.io/Intention-Timer/)
+
+ ## Future Additions
+Future additions to the Intention Timer will allow the users to be able to re-do specific cards, with the possibility to set different time frames and modify specifications. Besides the user will have the possibility to selected the most important and  relevant cards and save them as *Favorites*.
```

Comment:
Update second sentence to -
The user will also have the ability to select the most important and relevant cards and save them as *Favorites*.